src/runtime/text.h

```diff
@@ -87,6 +86,23 @@ static inline void push_character(string s, character c)
     }
 }
 
+/**
+ * Push an UTF-8 character from a sequence of bytes p and return the
+ * number of bytes consume from p.
+ */
+static inline int push_utf8_character(string s, const char *p)
+{
+    int nbytes = 0;
+    u32 crt_char = 0x00;
```

Comment:
There is no need to initialize crt_char here, since you are going to overwrite its value in the next line.

Reply:
You're right, it's a leftover from a previous version of this function :). I'll remove the initialization.